src/compas_fab/ghpython/artists.py

```diff
@@ -9,7 +9,7 @@
     from compas_ghpython.utilities import draw_mesh
     from compas_rhino.geometry.transformations import xtransform
 
-from compas_fab.artists import BaseRobotModelArtist
+from compas_fab.artists import BaseRobotModelArtist  # noqa: E402
```

Comment:
why does the linter complain? is it because there's code above? would it be possible to move this above instead of noqa'ing the warning?

Reply:
Yes, it's because of the code above.  I'll try to move it.